airbyte-integrations/connectors/source-asana/unit_tests/test_streams.py

```diff
@@ -30,3 +30,19 @@ def test_next_page_token():
     inputs = {"response": MagicMock()}
     expected = "offset"
     assert expected in stream.next_page_token(**inputs)
+
+
+@pytest.mark.parametrize(
+    ("http_status_code", "errors", "should_retry"),
+    [
+        (402, {}, False),
+        (403, {}, False),
+        (404, {}, False),
+        (451, {}, False),
```

Comment:
let's add a retriable status as well

Reply:
You mean "add another test case where we see the retry is triggered?"

Reply:
exactly :)

Reply:
Done.